bitrise.yml

```diff
@@ -64,8 +64,8 @@ workflows:
             echo "-- build-for-testing --"
             mkdir DerivedData
 
-            xcodebuild -resolvePackageDependencies -onlyUsePackageVersionsFromResolvedFile
-            xcodebuild "-project" "/Users/vagrant/git/firefox-ios/Client.xcodeproj" "-scheme" "Fennec" -configuration "Fennec" "CODE_SIGNING_ALLOWED=NO" -sdk "iphonesimulator" "-destination" "platform=iOS Simulator,name=iPhone 15,OS=17.5" "COMPILER_INDEX_STORE_ENABLE=NO" "build-for-testing" "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO" -derivedDataPath "/Users/vagrant/git/DerivedData" | xcpretty | tee xcodebuild_fennec.log
+            xcodebuild -resolvePackageDependencies -onlyUsePackageVersionsFromResolvedFile -skipPackagePluginValidation -skipMacroValidation -verbose
+            xcodebuild "-project" "/Users/vagrant/git/firefox-ios/Client.xcodeproj" "-scheme" "Fennec" -configuration "Fennec" "CODE_SIGNING_ALLOWED=NO" -sdk "iphonesimulator" "-destination" "platform=iOS Simulator,name=iPhone 15,OS=17.5" "COMPILER_INDEX_STORE_ENABLE=NO" "build-for-testing" "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO" -derivedDataPath "/Users/vagrant/git/DerivedData" -skipPackagePluginValidation -skipMacroValidation -verbose | xcpretty | tee xcodebuild_fennec.log
```

Comment:
@thatswinnie if you remove this part` | xcpretty | tee xcodebuild_fennec.log` from the xcodebuild command, all logs should appear and so you would be able to see the `SwiftLintBuildToolPlugin` part